web/app/forms.py

```diff
@@ -79,3 +80,10 @@ class TournamentPlayerForm(Form):
     dob = StringField("Date of Birth", validators=[Required()])
 
     submit = SubmitField()
+
+class MatchResultsForm(Form):
+    player_1_name = StringField("Black's Name")
```

Comment:
What if there's a typo in the name, or it gets submitted as a nickname, or multiple people have the same name? It would seem safer to submit the AGA ID, rather than the player's name, to have a better guarantee of finding a match. 

But perhaps there are issues with temporary AGA ids for day-of signups - I could be wrong about this approach.